esp/clients/ws_dfsclient/ws_dfsclient.cpp

```diff
@@ -330,6 +330,10 @@ class CServiceDistributedFileBase : public CSimpleInterfaceOf<INTERFACE>
     {
         UNIMPLEMENTED_X("CServiceDistributedFileBase::enqueueReplicate");
     }
+    virtual IUserDescriptor *queryUser() const override
+    {
+        UNIMPLEMENTED_X("CServiceDistributedFileBase::queryUser");
```

Comment:
This should pass through userDesc from the 874,19:     createLegacyDFSFile/lookupLegacyDFSFile functions and return it.